src/coreclr/vm/threads.cpp

```diff
@@ -1543,7 +1543,7 @@ void Thread::InitThread()
 {
     CONTRACTL {
         THROWS;
-        if (GetThreadNULLOk()) {GC_TRIGGERS;} else {DISABLED(GC_NOTRIGGER);}
+        if (GetThreadNULLOk()) {DISABLED(GC_NOTRIGGER);} else {GC_TRIGGERS;}
```

Comment:
```suggestion
        GC_NOTRIGGER;
```
I dot not see anything that triggers GC in the impl